packages/libs/core/tsconfig.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "sourceMap": false,
+    "removeComments": true
```

Comment:
Would this mean the standard Rollup bundle (unminified) will remove comments as well?

Reply:
Apparently yes, changed to false. Minification will remove them.